crates/matrix-sdk/src/event_cache/room/mod.rs

```diff
@@ -248,11 +248,9 @@ impl RoomEventCacheInner {
         for raw_event in account_data {
             match raw_event.deserialize() {
                 Ok(AnyRoomAccountDataEvent::FullyRead(ev)) => {
-                    // Sometimes the sliding sync proxy sends many duplicates of the read marker
+                    // Sometimes sliding sync sends many duplicates of the read marker
                     // event. Don't forward it multiple times to avoid clutter
                     // the update channel.
```

Comment:
We're not sure the native implementation does that, but my feeling is that we can remove this comment and keep the code as is, maybe with a much shorter comment that sounds less accusatory.